sockeye/model.py

```diff
@@ -45,6 +47,8 @@ class ModelConfig(Config):
     :param config_loss: Loss configuration.
     :param lexical_bias: Use lexical biases.
     :param learn_lexical_bias: Learn lexical biases during training.
+    :param weight_tying: Enables weight tying if True.
```

Comment:
Thats what I meant: also renaming the weight-tying parameter in the decoder. I think this would be the right thing to do and backwards compatibility should not stop us from doing this.